### PR TITLE
create contact: mela.md

### PR DESCRIPTION
**Pull request by AnnikaHel**

### CODE REVIEW

File: data/contact-persons/mela.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+    name: Mela Brandau
```
Comment (victoria-alissa) [inline data/contact-persons/mela.md:2]: Melanie

File: data/contact-persons/mela.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+    name: Mela Brandau 
+    level: Talent Acquisition
+    phone: 
+    mail: jobs@sinnerschrader.com
+    xing:
```
Comment (rneuhann) [inline data/contact-persons/mela.md:6]: https://www.xing.com/profile/Mela_Brandau/cv

File: data/contact-persons/mela.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+    name: Mela Brandau 
+    level: Talent Acquisition
+    phone: 
+    mail: jobs@sinnerschrader.com
+    xing: 
+    linkedin:
```
Comment (rneuhann) [inline data/contact-persons/mela.md:7]: https://www.linkedin.com/in/mela-brandau-aba453117/

**Review by rneuhann — DISMISSED** [review]

**Comment by rneuhann** [comment]
Jo sieht gut aus! @AnnikaHel

**Review by kotzendekrabbe — APPROVED** [review]

**Review by kotzendekrabbe — APPROVED** [review]

**Pull request merged**